### ISSUE TITLE
Implementation of a lot of essentials commands

### ISSUE THREAD

**Issue by Fyrsta** [TranxCraft/iTranxCraft#38]
Needs implementation of:
&bull; fly
&bull; gamemode
&bull; give
&bull; heal
&bull; item
&bull; kit
&bull; more
&bull; speed
&bull; time
&bull; balance
&bull; balancetop
&bull; eco
&bull; pay
&bull; sell
&bull; afk
&bull; help
&bull; ignore
&bull; list
&bull; msg
&bull; nick
&bull; r
&bull; rules
&bull; seen
&bull; suicide
&bull; whois
&bull; clearinventory
&bull; ext
&bull; gc
&bull; kick
&bull; kickall
&bull; kill
&bull; mute
&bull; ping
&bull; weather
&bull; setspawn
&bull; spawn
&bull; back
&bull; home
&bull; sethome
&bull; tp
&bull; tpa
&bull; tpaall
&bull; tpaccept
&bull; tpahere
&bull; tpall
&bull; tpdeny
&bull; tphere
&bull; tpo
&bull; tpohere
&bull; tptoggle

All other Essentials commands would have been specified in a different issue or won't be implemented.

(It might be better to just use Essentials, after seeing the size of this list)

**Comment by Wild1145** [comment]
Yeah... You may want to just modify essentials to your needs rather than re-create all of this, or if you do want to do it, do it over time like we are with Icarus and TFM...

**Comment by Fyrsta** [comment]
@Wild1145 This is an optional feature, so it can be done over any length of time or not at all. The reason for this and a lot of these optional issues is to reduce dependency on other plugins for the server. Having as much functionality in our plugin as possible makes it easier to manage and fix problems compared to having lots of plugins.

**Comment by Wild1145** [comment]
@WickedGamingUK True, I am just thinking that as Essentials commands are easy enough to load over, you could do it over a more extended period of time, and reduce the dependency on Essentials to none. That way you have full functionality all the time without needing to spend an extended amount of time re-writing essentials more or less, then when you have time, you can just crack on and add a few commands at a time.

**Comment by Fyrsta** [comment]
@Wild1145 That's a good idea. Implement essentials features over time to migrate them to being self-managed here whilst leaving everything else in-place.
